src/ui/server/controllers/insurance/business/broker/validation/rules/using-broker.ts

```diff
@@ -0,0 +1,32 @@
+import { ERROR_MESSAGES } from '../../../../../../content-strings';
+import FIELD_IDS from '../../../../../../constants/field-ids/insurance/exporter-business';
+import { RequestBody } from '../../../../../../../types';
+import { objectHasProperty } from '../../../../../../helpers/object';
+import generateValidationErrors from '../../../../../../helpers/validation';
+
+const {
+  BROKER: { USING_BROKER: FIELD_ID },
+} = FIELD_IDS;
+
+const {
+  EXPORTER_BUSINESS: { [FIELD_ID]: ERROR_MESSAGE },
+} = ERROR_MESSAGES.INSURANCE;
+
+/**
+ * validates using broker field
+ * checks if response has been provided
+ * @param {RequestBody} responseBody
+ * @param {object} errors
+ * @returns {object} errors
+ */
+const usingExporter = (responseBody: RequestBody, errors: object) => {
+  if (!objectHasProperty(responseBody, FIELD_ID)) {
+    const errorMessage = ERROR_MESSAGE.IS_EMPTY;
+
+    return generateValidationErrors(FIELD_ID, errorMessage, errors);
+  }
+
```

Comment:
We can make this simpler - added a new shared validation helper function recently, `emptyFieldValidation` - check out `first-name`. Can just call this helper functino passing the body, field ID and error message, so we can get the same functionality in one line